gtfs/spec/en/reference.md

```diff
@@ -612,6 +624,69 @@ Describes levels in a station. Useful in conjunction with `pathways.txt`, and is
 |  `level_index` | Float | **Required** | Numeric index of the level that indicates its relative position. <br><br>Ground level should have index `0`, with levels above ground indicated by positive indices and levels below ground by negative indices.|
 |  `level_name` | Text | Optional | Name of the level as seen by the rider inside the building or station.<hr>_Example: Take the elevator to "Mezzanine" or "Platform" or "-1"._|
 
+### location_groups.txt
+
+File: **Optional**
+
+Primary key (`location_group_id`, `location_id`)
+
+Defines location groups that indicate groups of stops where a rider may request pickup or drop off.
+
+| Field Name | Type | Required | Description |
+| ---------- | ---- | ------------ | ----------- |
+| `location_group_id` | ID | **Required** | Identifies a location group. ID must be unique across all `stops.stop_id`, locations.geojson `id`, and `location_groups.location_group_id` values. <br><br>A location group is a group of stops that together indicate locations where a rider may request pickup or drop off.<br><br>Multiple entries in `location_groups.txt` can have the same `location_group_id`. | 
```

Comment:
Wording here could be confusing. This says the location_group_id must be unique "across all values" in three files, but then says multiple records in this file (rows in this table) may have the same location_group_id. To create a useful group, of course multiple rows in this table must reference the same location_group_id (the key for an implicit location_group entity that is not materialized in a table) but the keys of those implicit location_groups share a namespace with the primary keys of two other tables. The situation is unusual enough that wording will be important to avoid confusing people with more conventional data modeling expectations.